builtin/providers/google/resource_compute_instance_group_manager.go

```diff
@@ -144,6 +144,7 @@ func resourceComputeInstanceGroupManagerCreate(d *schema.ResourceData, meta inte
 		BaseInstanceName: d.Get("base_instance_name").(string),
 		InstanceTemplate: d.Get("instance_template").(string),
 		TargetSize:       target_size,
+		ForceSendFields:  []string{"Name", "BaseInstanceName", "InstanceTemplate", "TargetSize"},
```

Comment:
This is required, otherwise `TargetSize: 0` will be dropped when request is made to the Google API.
See https://github.com/google/google-api-go-client/issues/197 for the detail.